### ISSUE TITLE
dev-server no longer connects when using Node 18

### ISSUE THREAD

**Issue by xXBJXx** [ioBroker/dev-server#238]
I tried the new Node 18 version and found out that the dev-server doesn't connect at all anymore, there is always only this:
``` shell
[HPM] Error occurred while proxying request localhost:8081/ to http://localhost:20426/ [ECONNREFUSED] (https://nodejs.org/api/errors.html#errors_common_system_errors)
[HPM] Error occurred while proxying request localhost:8081/favicon.ico to http://localhost:20426/ [ECONNREFUSED] (https://nodejs.org/api/errors.html#errors_common_system_errors)
```
As soon as I switch back to node 16 (LTS), it connects again as before.
There are no other error messages only what I posted above.

**Comment by Apollon77** [comment]
maybe affected by the breaking change wit network family in nodejs 18?

**Comment by Apollon77** [comment]
BTW this breaking change could be reverted in nodejs 18.2 ... see https://github.com/nodejs/TSC/pull/1227/files#diff-e5b3ef0c3cff74b889d643255af6994b13341e5ffd53f405109a72f7863b9c56R36

**Comment by Apollon77** [comment]
But we stilkl see dns changes... so localhost is sometimes looked up to the IPv6 adress but it only listens to ipv4 ...

**Comment by Apollon77** [comment]
fix in 0.6.0
